### DISCUSSION TITLE
Step 6 script `Build APK with Flutter` exited with status code 1

### DISCUSSION THREAD

**Discussion by dnbucket88** [codemagic-ci-cd/codemagic-docs#2237] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

![12345](https://github.com/codemagic-ci-cd/codemagic-docs/assets/151423167/a33020ec-3733-42db-9fae-285d3affd792)
![123456](https://github.com/codemagic-ci-cd/codemagic-docs/assets/151423167/c7e58795-fcd4-4b13-855d-a79c2f0524aa)

facing these errors. I am using my a website https://siteintoapp.com to try create it. Site is created in php.. Plz help

### Expected results

To build fully working app

### Actual results

Step 6 script `Build APK with Flutter` exited with status code 1

**Comment by dnbucket88** [comment]
I am Still looking for a solution.

**Reply by dtrdic** [reply]
Hi @dnbucket88, not sure what you are exactly doing since you haven't provided a build ID. 
If you want to publish a web app to a custom subdomain for testing, use a custom subdomain of codemagic.app and not your website. Check out [this docs](https://docs.codemagic.io/flutter-publishing/publishing-to-codemagic-static-pages/)

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion due to a lack of activity. You can open the discussion again by replying to this message or opening a new discussion.
